lib/graphql/directive.rb

```diff
@@ -8,10 +8,12 @@ module GraphQL
   #
   class Directive
     include GraphQL::Define::InstanceDefinable
-    accepts_definitions :locations, :name, :description, :arguments, argument: GraphQL::Define::AssignArgument
+    accepts_definitions :locations, :name, :description, :arguments, :default, argument: GraphQL::Define::AssignArgument
```

Comment:
For the sake of readability, what do you think about calling this `default_directive`? It has more symmetry with the rest of these changes and seeing things like `default = false` in isolation might be confusing.